pkg/minikube/cruntime/docker.go

```diff
@@ -272,15 +271,19 @@ func (r *Docker) SystemLogCmd(len int) string {
 // 3. Remove the tarball within the VM
 func (r *Docker) Preload(k8sVersion string) error {
 	tarballPath := download.TarballPath(k8sVersion)
-	dest := "/preloaded.tar.lz4"
+	targetDir := "/"
+	targetName := "preloaded.tar.lz4"
+	// don't use filepath.Join so that we maintain Linux file separators
+	// so that this still works on Windows
+	dest := targetDir + targetName
```

Comment:
you can use path.Join instead of filepath.Join here

Reply:
done